drizzle-orm/src/mysql-core/indexes.ts

```diff
@@ -99,3 +109,7 @@ export function index(name: string): IndexBuilderOn {
 export function uniqueIndex(name: string): IndexBuilderOn {
 	return new IndexBuilderOn(name, true);
 }
+
+export function fulltextIndex(name: string): IndexBuilderOn {
+	return new IndexBuilderOn(name, true);
```

Comment:
This line is passing true to the unique parameter. Yet, in the IndexBuilder constructor it throws an error if fulltext and unique are both true. It doesn't seem to make sense.